src/gt4py/gt_src_manager.py

```diff
@@ -28,29 +29,33 @@
 
 
 def install_gt_sources() -> bool:
-    is_ok = has_gt_sources()
-    if not is_ok:
-        GIT_BRANCH = _GRIDTOOLS_GIT_BRANCH
-        GIT_REPO = _GRIDTOOLS_GIT_REPO
+    if has_gt_sources():
+        return True
```

Comment:
I made this an early return instead of indenting the entire function

Reply:
This is generally considered an antipattern, since it's easy to miss the fact that the function returns early when scanning code. If you want to avoid indenting a lot of code, one way would be to make this function 2/3 lines (`if not has_gt_sources(): _force_install_gt_sources()`) and put most of the logic in a helper function.

Reply:
Good to know. It's fairly short after refactoring, so I just inlined it.